scripts/run_e2e_suites_for_cov.sh

```diff
@@ -0,0 +1,38 @@
+
+export COVERAGE_MODE=true
+export LEDGER_SIM=true
+
+rm -rf coverage/e2e
```

Comment:
the provided script was just an example, this line for example needs PWD, and before that, a check that PWD is correct needs to be done (similar to other scripts)